college_preprocessor.py

```diff
@@ -89,32 +188,68 @@ def parsecolleges(college_file, columns):
     adjusts SAT/ACT & student/faculty ratio to contain
     real valued numbers for ML algorithms
     '''
-    df = pd.read_csv(college_file, sep='\t', header=None)
+
+    # read the csv
+    df = pd.read_csv(college_file, header=None)
     df.columns = columns
-    df['SAT1'] = df['SAT/ACT 25th-75th percentile'].apply(SAT1)
-    df['SAT2'] = df['SAT/ACT 25th-75th percentile'].apply(SAT2)
-    df['ACT1'] = df['SAT/ACT 25th-75th percentile'].apply(ACT1)
-    df['ACT2'] = df['SAT/ACT 25th-75th percentile'].apply(ACT2)
-    df['Student-faculty ratio'] = df['Student-faculty ratio'].apply(ratio)
+
+    # change all 'N/A' values to 0
+    df.fillna(0, inplace=True)
```

Comment:
If 0 is the default value for N/A then it should not associated with another value like 'No' or 'least-selective'